### PR TITLE
Adding missing exports to index.ts

### PR DESCRIPTION
**Pull request by ericwill-msft**

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -42,7 +42,11 @@ export {
   LiveResponse,
   LiveValidationIssue,
   LiveValidatorOptions,
-  ApiOperationIdentifier
+  ApiOperationIdentifier,
+  RequestValidationResult,
+  ResponseValidationResult,
+  ValidateOptions,
+  formatUrlToExpectedFormat
```
Comment (vladbarosan) [inline index.ts]: I don't think we want to export "formatUrlToExpectedFormat" thats a function internal to the module

File: index.ts [hunk 2]
```diff
@@ -42,7 +42,11 @@ export {
...
 } from "./lib/validators/liveValidator"
 export { SpecResolver } from "./lib/validators/specResolver"
```
Comment (vladbarosan) [inline index.ts]: also lets update the `package.json` patch version otherwise publishing will fail

Comment (ericwill-msft) [reply index.ts]: Updated to 0.15.4

File: index.ts (hunk 1 above)
Comment (ericwill-msft) [reply index.ts]: Removed

**Pull request merged**